### PR TITLE
Update mobile-optimize.md for Cosmos

### PR DESCRIPTION
**Pull request by rsrbk**

### CODE REVIEW

File: dapps/mobile-optimize.md [hunk 1]
```diff
@@ -26,20 +26,20 @@ iOS users require the latest build from Apple App Store. UI/UX design needs to a
 
 ## Master Web3
 
-Web3 is incredibly powerful for developing DApps and outlining their interactions with Ethereum or Solana nodes. The most up-to-date Web3 open-source [Ethereum JavaScript library](https://github.com/ethers-io/ethers.js/) or [Solana JavaScript library](https://solana-labs.github.io/solana-web3.js/modules.html) needs to be implemented for your DApp to be well optimized for the Trust Wallet browser.
```
Comment (rsrbk) [inline dapps/mobile-optimize.md:29]: I feel like this paragraph is not relevant

Comment (hewigovens) [reply dapps/mobile-optimize.md:29]: Move to a bottom "Reference" section? suggested libraries

**Review by hewigovens — APPROVED** [review]
Looks good, don't merge it until it's released

File: dapps/mobile-optimize.md (hunk 1 above)
Comment (ArtemGoryunov) [reply dapps/mobile-optimize.md:29]: good idea

**Review by ArtemGoryunov — APPROVED** [review]

**Pull request merged**